webhooks/virtualmachine/v1alpha2/validation/virtualmachine_validator.go

```diff
@@ -336,6 +336,7 @@ func (v validator) validateNetwork(ctx *context.WebhookRequestContext, vm *vmopv
 			defaultNetworkInterface.Name = "eth0"
 		}
 		allErrs = append(allErrs, v.validateNetworkInterfaceSpec(networkPath, defaultNetworkInterface)...)
+		allErrs = append(allErrs, v.validateNetworkSpecWithBootstrap(networkPath, defaultNetworkInterface, vm)...)
```

Comment:
FYI - when you rebase the top-level network interface related fields are now removed